src/components/menu/templates/enterprise.tsx

```diff
@@ -12,7 +12,8 @@ const items = {
   'menu-enterprise-authentication-security': {
     'Auth Connect': '/docs/enterprise/auth-connect',
     '': {
-      'Auth0': '/docs/enterprise/auth-connect/auth0'
+      'Auth0': '/docs/enterprise/auth-connect/auth0',
+      'Azure AD': '/docs/enterprise/auth-connect/azure-ad'
```

Comment:
we might want to call this Azure AD B2C, not v2.

Reply:
ah good call. done.